sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/command.py

```diff
@@ -724,6 +727,19 @@ def __call__(self, *args, **kwargs) -> "Command":
             error_type=ValidationErrorType.INVALID_VALUE,
         )
 
+    @classmethod
+    def _resolve_pipeline_input_for_node_attrs(cls, node_attr: object):
+        """Resolve PipelineInput for various node attributes"""
+        if node_attr is None:
+            return None
+        if isinstance(node_attr, CommandJobLimits):
+            timeout_value = node_attr.timeout
+            if isinstance(timeout_value, PipelineInput):
+                # call result() to get real value for PipelineInput object
+                node_attr.timeout = timeout_value.result()
```

Comment:
Is this a temporary work around before backend supports the binding?

Reply:
Removed the part, keep binding string as we discussed offline.